### DISCUSSION TITLE
Removing React Favicon

### DISCUSSION THREAD

**Discussion by Gabrielle-Tobermann** [nss-evening-cohort-14/evening-client-side#267] [Help Me!] [answered]
Gabby Tobermann 

Other tickets? No. 

Issue: I have been trying to remove the react favicon from my portfolio site but it's just keeps appearing. I managed to get rid of it on my local, but it's still there on the deployed link. 

Google links: 
https://github.com/facebook/create-react-app/issues/4597
https://stackoverflow.com/questions/51994375/favicon-in-react-will-not-update

I have tried: removing  the link to the favicon in the index file, adding it back but adding 'v=2' at the end (as suggested in one of the links). Hard refreshed, emptied cache. It went way after that on my local server, but then when I merged, my deployed link still had it. 

Code: 
https://github.com/Gabrielle-Tobermann/react-personal-bio-site/blob/main/public/index.html

**Comment by drteresavasquez** [comment]
So, you have a manifest that links to all the logos. You need to update the files in the public directory with the logos to make this happen. Let me know if you want to look at it together.

**Reply by Gabrielle-Tobermann** [reply]
Actually whenever you have minute, I might need some help. I've updated the files but it's not updating. No rush though, I know you are busy approving planning.

**Reply by ajawashington** [reply]
@Gabrielle-Tobermann I will come to your room to help.

**Comment by Gabrielle-Tobermann** [comment, marked as answer]
Solution: in the link to the icon I had .png but it was a jpg file 🤦‍♀️
